docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -507,86 +529,100 @@ Perform the following steps as part of the cutover process:
+    Refer to [cutover status](../../reference/cutover-archive/cutover/#cutover-status) for details about the arguments.
 
-1. Setup indexes and triggers to the fall-forward database manually. Also, re-enable the foreign key and check constraints.
+1. Setup indexes and triggers to the source-replica database manually. Also, re-enable the foreign key and check constraints.
 
-1. Verify your migration. After the schema and data import is complete, the automated part of the database migration process is considered complete. You should manually run validation queries on both the source and fall-forward databases to ensure that the data is correctly migrated. A sample query to validate the databases can include checking the row count of each table.
+1. Verify your migration. After the schema and data import is complete, the automated part of the database migration process is considered complete. You should manually run validation queries on both the source and source-replica databases to ensure that the data is correctly migrated. A sample query to validate the databases can include checking the row count of each table.
 
-    {{< warning title = "Caveat associated with rows reported by import data status" >}}
+    {{< warning title = "Caveat associated with rows reported by get data-migration-report" >}}
 
 Suppose you have a scenario where,
 
-- [import data](#import-data) or [import data file](../bulk-data-load/#import-data-files-from-the-local-disk) command fails.
+- [import data to source-replica](#import-data-to-source-replica) command fails.
 - To resolve this issue, you delete some of the rows from the split files.
-- After retrying, the import data command completes successfully.
+- After retrying, the import data to target command completes successfully.
 
-In this scenario, [import data status](#import-data-status) command reports incorrect imported row count; because it doesn't take into account the deleted rows.
+In this scenario, [get data-migration-report](#get-data-migration-report) command reports incorrect imported row count; because it doesn't take into account the deleted rows.
 
 For more details, refer to the GitHub issue [#360](https://github.com/yugabyte/yb-voyager/issues/360).
 
     {{< /warning >}}
 
-1. Stop [archive changes](#archive-changes-optional).
+### End migration
 
-{{< note >}}
-During `fall-forward synchronize`, yb-voyager creates a CDC stream ID on the target YugabyteDB database to fetch the new changes from the target database which is displayed as part of the `fall-forward synchronize` output. You need to manually delete the stream ID after `fall forward switchover` is completed.
-{{< /note >}}
+To end the migration, you need to clean up the export directory (export-dir), and Voyager state ( Voyager-related metadata) stored in the target YugabyteDB database and source-replica database.
```

Comment:
not the source as well?

Reply:
Yes it should be there I think. @priyanshi-yb leaving this open for you to confirm . I'm making the change though.

Reply:
No source should not be there in case of fall-forward. Actually Voyager state is only created if there is any import operation on the database but in case of fall-forward there is no import operation on source hence no cleanup is required

Reply:
@aishwarya24 pls rever the change